notebook/notebookapp.py

```diff
@@ -223,6 +223,7 @@ def init_settings(self, jupyter_app, kernel_manager, contents_manager,
             base_url=base_url,
             default_url=default_url,
             template_path=template_path,
+            use_redirect_file=jupyter_app.use_redirect_file,
```

Comment:
I don't think the tornado web application object needs this - it's only used in the NotebookApp class. So this line can go.

Reply:
Thanks for the quick review. The change should be reflected in the newest file version now.